tests/transports-integrations/tests/integrations/test_google.py

```
@@ -0,0 +1,472 @@
+                        "name": tool["name"],
+                        "description": tool["description"],
+                        "parameters": tool["parameters"],
+                    }
+                ]
+            }
+        )
+
+    return google_tools
+
+
+def load_image_from_url(url: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `load_image_from_url`

(ANN201)